config/sudoers-for-dev

```diff
@@ -30,6 +30,8 @@ vx-admin ALL=(root:ALL) NOPASSWD: /vx/admin/admin-functions/program-system-admin
 vx-admin ALL=(root:ALL) NOPASSWD: /usr/local/bin/tpm2-totp
 vx-admin ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/mount-usb.sh
 vx-admin ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/unmount-usb.sh
+vx-admin ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/auth/src/create-cert
+vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/auth/src/create-cert
```

Comment:
Do both of these users need to be able to run these scripts?

Reply:
Yeah, thanks for checking! They do - the explanation is tucked away in the commit message:

> \- vx-admin user needs this for sys admin card programming in configuration wizard
> \- vx-services user needs this for card programming in general